packages/components/components/elvis-dropdown/src/react/elvia-dropdown.tsx

```diff
@@ -206,7 +208,10 @@ export const Dropdown: React.FC<DropdownProps> = ({
         aria-haspopup="true"
       >
         {!!label && <FormFieldLabel>{label}</FormFieldLabel>}
-        <DropdownInputContainer ref={connectedElementRef}>
+        <FormFieldInputContainer
+          style={{ width: '100%', cursor: isDisabled ? 'default' : 'pointer' }}
```

Comment:
I see that it is noting new; but isn't it strange how the `cursor` becomes `default` instead of `not-allowed` when disabled? I cant find any design on this, so not sure if this is a bug or not

Reply:
We can probably change this, yes 🙂